composables/useDataByGrainType.ts

```diff
@@ -1,8 +1,13 @@
 import { group } from 'd3'
 
-export async function useDataByGrainType() {
-  const grainData = await useGrainData();
-  // const groupedData = group(grainData, (d) => d.metadata[0][0]);
+export function useDataByGrainType() {
+  const grainData = useGrainData();
   const groupedData = group(grainData, (d) => d.fileName);
-  return groupedData
+  // remove groupings where the key includes the word "dynamics"
+  groupedData.forEach((value, key) => {
```

Comment:
Why do we do this twice?

Reply:
ie it also happens in `composables/useGrainData.ts` - shouldn't it only happen once at the top-level loading?

Reply:
yes- i have a better idea how we should do this.

Reply:
instead of scaling all the data, we should just scale the activeData as a final step. it's less computation and means we only would need to group the data once.

Reply:
Cool, let's do that and :shipit: